src/main/webapp/aboutus.html

```diff
@@ -22,11 +22,27 @@
     <link rel="stylesheet" href="/css/main.css">
```

Comment:
Make sure the pull request description has more details on the changes you are making.